src/main/java/com/piekoszek/nowaksharedrent/auth/AuthController.java

```diff
@@ -31,8 +30,17 @@
     ResponseEntity<Object> login(@RequestBody Account account) {
         Optional<String> token = authService.loginUser(account);
         if (token.isPresent()) {
-            return new ResponseEntity<>(new TokenResponse(token.get()), HttpStatus.CREATED);
+            return new ResponseEntity<>(new TokenResponse(token.get()), HttpStatus.OK);
         }
         return new ResponseEntity<>(new MessageResponse("Invalid email or password!"), HttpStatus.UNAUTHORIZED);
     }
+
+    @GetMapping("/auth/refresh")
+    ResponseEntity<Object> refreshToken(@RequestHeader("Authorization") String oldToken) {
```

Comment:
refreshToken might be confusing, https://auth0.com/blog/refresh-tokens-what-are-they-and-when-to-use-them/
maybe rename to updateTokenData, or sth like that